### DISCUSSION TITLE
Where can I find the manual of the latest version for “dpgen autotest"?

### DISCUSSION THREAD

**Discussion by THBXM** [deepmodeling/dpgen#586] [Q&A]
The parameters file setting of "dpgen autotest" in versions above 0.10.0 have changed a lot, but I can't find the relevant documents and instructions

**Comment by AnguseZhang** [comment]
https://github.com/deepmodeling/dpgen/wiki/Auto-test

**Reply by THBXM** [reply]
Thanks!
